package.json

```diff
@@ -104,7 +104,7 @@
     "@babel/core": "7.4.5",
     "@babel/polyfill": "7.4.4",
     "@babel/register": "7.4.4",
-    "@elastic/charts": "^7.2.1",
+    "@elastic/charts": "^8.0.1",
```

Comment:
requires changes from https://github.com/elastic/elastic-charts/pull/304. Linked locally for development.